packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -101,6 +101,7 @@ class SocksProxyConnection {
     const tlsOptions: tls.ConnectionOptions = {
       socket: this.target,
       rejectUnauthorized: !this.socksProxy.ignoreHTTPSErrors,
+      servername: this.host,
```

Comment:
Can this.host be an IP address? Do we validate it elsewhere?

Reply:
Its coming from socks, already split up / validated.